### PR TITLE
MAINT: Remove unnecessary 'from __future__ import ...' statements

### PR DESCRIPTION
**Pull request by jdufresne**
As numpy is Python 3 only, these import statements are now unnecessary and don't alter runtime behavior.

### CODE REVIEW

File: tools/openblas_support.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-from __future__ import division, absolute_import, print_function
-import os
```
Comment (mattip) [inline tools/openblas_support.py]: do not remove the `os` import

**Comment by jdufresne** [comment]
Rebased for 2020 :tada:

**Comment by jarrodmillman** [comment]
@eric-wieser This seems like a good PR to merge next.  It only removes `__future__` imports.  I scanned all the changes and it looks good to me and it looks like it got them all.

**Comment by seberg** [comment]
I agree, will merge after tests pass. The comment about the doc-test not being runnable confused me, but its 13 years old, so I think its just a left over from a different age.

**Comment by seberg** [comment]
Thanks @jdufresne  and @jarrodmillman!

**Pull request merged**